js/src/alert.js

```diff
@@ -111,7 +94,19 @@ class Alert extends BaseComponent {
  * ------------------------------------------------------------------------
  */
 
-EventHandler.on(document, EVENT_CLICK_DATA_API, SELECTOR_DISMISS, Alert.handleDismiss(new Alert()))
+EventHandler.on(document, EVENT_CLICK_DATA_API, SELECTOR_DISMISS, function (event) {
+  if (['A', 'AREA'].includes(this.tagName)) {
+    event.preventDefault()
+  }
+
+  if (isDisabled(this)) {
+    return
+  }
```

Comment:
:warning: This changes the behavior

Reply:
Is a behavior that we support on the other components. Right?
As by default dev will not use anchor elemens , either disabled (doc doesn't propose something )

Reply:
That's good though, but I just wanted to mention that it was not before :slightly_smiling_face: